sale_order_type/models/sale.py

```diff
@@ -80,7 +80,14 @@ def _compute_warehouse_id(self):
                 order.warehouse_id = order_type.warehouse_id
         return res
 
-    @api.depends("type_id")
+    def _depends_picking_policy(self):
```

Comment:
Can you bump the version number of this module?  As a patch version. So we can merge with `nobump` option.

Reply:
Sure, will do

Reply:
Thanks!